src/main/kotlin/no/nav/henvendelsesarkiv/Database.kt

```diff
@@ -0,0 +1,93 @@
+package no.nav.henvendelsesarkiv
+
+import no.nav.henvendelsesarkiv.model.Arkivpost
+import no.nav.henvendelsesarkiv.model.ArkivpostMapper
+import no.nav.henvendelsesarkiv.model.Vedlegg
+import no.nav.henvendelsesarkiv.model.VedleggMapper
+import org.springframework.jdbc.core.JdbcTemplate
+
+private const val ARKIVPOST_SQL = """
+            INSERT INTO arkivpost(arkivpostId, arkivertDato, mottattDato, utgaarDato, temagruppe, arkivpostType, dokumentType,
+                                  kryssreferanseId, kanal, aktoerId, fodselsnummer, navIdent, innhold, journalfoerendeEnhet,
+                                  status, kategorikode, signert, erOrganInternt, begrensetPartInnsyn, sensitiv
+            ) VALUES (
+                ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?
```

Comment:
Nå spør jeg bare fordi jeg er nub, men hva er alle disse spørsmålstegnene? :P

Reply:
String-substitutter. Når denne benyttes sendes det med verdier for alle disse feltene.

Reply:
SQL prepared statements, der man bruker "?" som substitutt og skyter inn verdier, det er 3 gode grunner til å bruke det:
- Litt mer ryddig enn å bygge SQL med masse string-skjøting
- Ulike SQL-databaser tolker sql på litt ulike måter, med prepared-statement så fikser driveren det sånn at verdier blir riktig formattert
- SQL-en din er ikke sårbar for SQL-injection